INSTALL.md

```diff
@@ -387,6 +388,24 @@ autoreconf -if
 make -j$(nproc)
 sudo make install
 cd ..
+
+
+Conversely, if you wish to explictly statically  or dynamically link sqlcipher use:
```

Comment:
remove unneeded whitespace;

Why would `Conversely, ` part be needed? It can start just from `If`